bundles/edu.kit.kastel.sdq.eclipse.grading.view/src/edu/kit/kastel/eclipse/grading/view/assessment/ArtemisGradingView.java

```diff
@@ -81,6 +86,14 @@ private void addListenerForMarkerDeletion() {
 				}));
 	}
 
+	private void initializeAnnotationEditing() {
+		IViewPart view = PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage()
+				.findView("edu.kit.kastel.eclipse.common.view.marker.AssessmentMarkerView");
```

Comment:
We should move that to constants as well

Reply:
What about `AssessmentMarkerView.class.getName()`. This would allow future refactoring without need to edit the constant at all?

Reply:
Sounds good.